backend/src/test/java/com/phaete/backend/forage/service/ForageMapItemServiceTest.java

```diff
@@ -0,0 +1,336 @@
+
+	private final ForageMapItemRepository forageMapItemRepository = mock(ForageMapItemRepository.class);
+	private final IdService idService = mock(IdService.class);
+
+
+	@Test
+	void createForageMapItem_expectDTO_onSuccess() {
+		ForageMapItemDTO expectedForageMapItemDTO = new ForageMapItemDTO(
+				new ForageWikiItem(
+						"1",
+						"Apple Tree",
+						ForageCategory.FRUIT,
+						ForageSource.TREE,
+						"Apple Tree",
+						ForageSeason.FALL,
+						List.of("test")
+				),
+				new CustomMarker(
+						"1",
+						new double[] {0.0, 0.0},
+						"",
+						new int[] {0, 0},
+						new int[] {0, 0},
+						new int[] {0, 0},
+						""
+				),
+				ForageQuantity.ABUNDANT,
+				ForageQuality.EXCELLENT,
+				"never",
+				"notes"
+		);
+
+		when(forageMapItemRepository.save(any(ForageMapItem.class))).thenReturn(
+				new ForageMapItem(
+						"1",
+						new ForageWikiItem(
+								"1",
+								"Apple Tree",
+								ForageCategory.FRUIT,
+								ForageSource.TREE,
+								"Apple Tree",
+								ForageSeason.FALL,
+								List.of("test")
+						),
+						new CustomMarker(
+								"1",
+								new double[] {0.0, 0.0},
+								"",
+								new int[] {0, 0},
+								new int[] {0, 0},
+								new int[] {0, 0},
+								""
+						),
+						ForageQuantity.ABUNDANT,
+						ForageQuality.EXCELLENT,
+						"never",
+						"notes"
+				)
+		);
+		when(idService.generateId()).thenReturn("1");
+
+		ForageMapItemService forageMapItemService = new ForageMapItemService(
+				forageMapItemRepository,
+				new ConverterService(idService)
+		);
+
+		ForageMapItemDTO actualForageMapItemDTO = forageMapItemService.createForageMapItem(expectedForageMapItemDTO);
+		verify(forageMapItemRepository).save(any(ForageMapItem.class));
+		assertEquals(expectedForageMapItemDTO, actualForageMapItemDTO);
+	}
+
+	@Test
+	void findAllForageMapItems_expectList_onSuccess() {
+		List<ForageMapItem> expectedForageMapItems = List.of(
+				new ForageMapItem(
+					"1",
+					new ForageWikiItem(
+							"1",
+							"Apple Tree",
+							ForageCategory.FRUIT,
+							ForageSource.TREE,
+							"Apple Tree",
+							ForageSeason.FALL,
+							List.of("test")
+					),
+					new CustomMarker(
+							"1",
+							new double[] {0.0, 0.0},
+							"",
+							new int[] {0, 0},
+							new int[] {0, 0},
+							new int[] {0, 0},
+							""
+					),
+					ForageQuantity.ABUNDANT,
+					ForageQuality.EXCELLENT,
+					"never",
+					"notes"
+				)
+		);
+		when(forageMapItemRepository.findAll()).thenReturn(expectedForageMapItems);
+
+		ForageMapItemService forageMapItemService = new ForageMapItemService(
+				forageMapItemRepository,
+				new ConverterService(idService)
+		);
+
+		List<ForageMapItem> actualForageMapItems = forageMapItemService.findAllForageMapItems();
+		verify(forageMapItemRepository).findAll();
+		assertEquals(expectedForageMapItems, actualForageMapItems);
+	}
+
+	@Test
+	void findAllForageMapItems_expectList_WithoutInvalidWikiItem() {
```

Comment:
I don't understand what the tested method is expected to do in this test case.
Renaming might help.

Reply:
maybe introducing variables for what you call "Invalid Items" might also help to understand what is invalid and what should happen to invalid items.